### ISSUE TITLE
An easier way to obtain the app token

### ISSUE THREAD

**Issue by Stvad** [felixrieseberg/slack-archive#4]
If you were to click "install to workspace" button on the app page - it'll give you the token you can use

More detailed steps are described in https://github.com/sebseager/slack-exporter#authentication-with-slack

**Comment by bellomia** [comment]
Yes, furthermore **the provided way to get the token appears to be currently broken**. The browser never gives the token at the end, returning instead an error message for "invalid code".

The new beta option that allows creating a new app through an "App Manifest" seems reliable instead and all the configs could be easily provided through a yaml script, instead of making the user add the scopes one by one in the standard "type and click" interface.

I put everything in an update of the README file in my fork, also the yaml configs, along the lines of the reference provided by @Stvad. You may consider to do something similar here (or we can arrange a pull request, but keep in mind that I also added many minor changes here and there in the README, according to my taste, so you may want to avoid merging everything).

**Comment by bellomia** [comment]
Anyway thanks @Stvad! 

Having this alternate way pinned here really saved me: I was in a rush cause I had access to all my Slack data for a very narrow timeframe (random trial period in a non pro workspace, just about to end in a few hours), and I really wanted to get my nice html backup, but the authorization kept failing and... well I'd say that it felt almost like the classic movie set, with all the unreal "fast-hacking" scenes 😅
